docs/changelog/1_0_a3.rst

```diff
@@ -0,0 +1,158 @@
+===========
+1.0 Alpha 3
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 3.
+
+
+Rust
+====
+
+We've started the process of rewriting some of the
+performance-critical EdgeDB internals to Rust. So far we have
+converted the lexer as well as the GraphQL parser.
+
+Additionally, we're also making EdgeDB command-line tools using Rust.
```

Comment:
making...using --- this sentence needs a rewrite. I'd also mention the command line tools first and then talk about graphql.

Reply:
OK, I'll swap order.